### ISSUE TITLE
Unable to Setup Configuration

### ISSUE THREAD

**Issue by helloitisjc** [eneiluj/integration_suitecrm#3]
I'm 100% sure it's my end of things here, though I've followed the SuiteCRM documentation to create the OpenSSL items as mentioned, I've created the secret manually and transferred that associated ID over to Nextcloud in the admin settings, but I can't seem to get it to log in. It continually tells me invalid login when I put in my user login information, even though I know that is 100% correct. Any insight/help would be appreciated.

**Comment by a5pire** [comment]
Yep, likewise... I'm 100% sure my set up is correct and all details are correct but I'm continually presented with a 500 internal server error in the logs..... Disabled the plugin. I doubt its being maintained given the last commits.

**Comment by Ozzie76** [comment]
> I'm 100% sure it's my end of things here, though I've followed the SuiteCRM documentation to create the OpenSSL items as mentioned, I've created the secret manually and transferred that associated ID over to Nextcloud in the admin settings, but I can't seem to get it to log in. It continually tells me invalid login when I put in my user login information, even though I know that is 100% correct. Any insight/help would be appreciated.

@helloitisjc I ran into the same and got it going. Posted here what worked for me: https://github.com/eneiluj/integration_suitecrm/issues/2#issuecomment-1030570086
